firebase_admin/_messaging_utils.py

```diff
@@ -588,6 +603,20 @@ def encode_webpush_notification_actions(cls, actions):
             results.append(cls.remove_null_values(result))
         return results
 
+    @classmethod
+    def encode_webpush_fcm_options(cls, options):
+        """Encodes a WebpushFcmOptions instance into JSON."""
+        if options is None:
+            return None
+        result = {
+            'link': _Validators.check_string('WebpushConfig.fcm_options.link', options.link),
+        }
+        result = cls.remove_null_values(result)
+        link = result.get('link')
+        if link is not None and not link.startswith('https://'):
```

Comment:
Should we allow alternate capitalization on the protocol? I mean, "HTTPS://" is probably okay, right?

Reply:
Yes, but we have always checked only for the lower-cased prefix in this codebase (there are similar URL checks in auth, db etc). So I'll keep this consistent with that. Also developers haven't  requested this, so it should be ok.